webapp/pom.xml

```diff
@@ -94,7 +94,7 @@
          <plugin>
             <groupId>org.mortbay.jetty</groupId>
             <artifactId>maven-jetty-plugin</artifactId>
-            <version>6.1.10</version>
+            <version>6.1.26</version>
```

Comment:
I vote for merging this with a 6.x version of Jetty. I presume Ryan will want to backport this to a released maintenance line.

Then in another ticket (that won't be backported) we can go to Jetty 8.x.